phone_book.py

```diff
@@ -0,0 +1,35 @@
+book = dict()
+a = True
+while a:
```

Comment:
nit: can be written in one line, there is no reason to set it to variable `a`
```suggestion
while True:
```